README.md

```diff
@@ -73,6 +73,12 @@ options for the offset in the file/text provided:
 ]
 ```
 
+To use the iOS SDK is to specified the `compilerargs` option. The value
```

Comment:
How about just saying this:

> Use the `compilerargs` argument to pass compiler arguments to SourceKit:

And is the `--` correct here? Normally, the convention is to treat everything past the `--` as a variable-length array of values, which is not the case here since the compiler arguments are given as a single string.